pkg/apis/kops/cluster.go

```diff
@@ -180,7 +180,8 @@ type Assets struct {
 
 // IAMSpec adds control over the IAM security policies applied to resources
 type IAMSpec struct {
-	Legacy bool `json:"legacy"`
+	Legacy   bool `json:"legacy"`
+	AllowECR bool `json:"allowECR,omitempty"`
```

Comment:
How about allowContainerRegistry? Someone more cloud generic?

Reply:
Sure sounds good, I've just updated this.